src/java.base/share/classes/sun/reflect/annotation/AnnotationInvocationHandler.java

```diff
@@ -490,7 +492,20 @@ private void validateAnnotationMethods(Method[] memberMethods) {
          * 9.6.1. Annotation Type Elements
          */
         boolean valid = true;
+        Method currentMethod = null;
         for(Method method : memberMethods) {
+            currentMethod = method;
```

Comment:
I can't see any use of currentMethod, am I missing something?

Reply:
Remnant of a second fix I pulled out for separate handling; I'll remove before pushing. Thanks.